packages/stable/src/__tests__/api/datamodels.int.spec.ts

```diff
@@ -121,6 +125,31 @@ describe('Data models integration test', () => {
     ]);
     expect(datamodel.items.length).toBe(1);
     expect(datamodel.items[0].name).toEqual(datamodelCreationDefinition.name);
+    expect(datamodel.items[0].views).toBeDefined();
```

Comment:
i thought without `inlineViews: true`, this would be undefined? or should we check that it always returns an empty array?

Reply:
aha - so we could add here that e.g. properties should be undefined?

Reply:
added